body/stretch_body/hello_utils.py

```diff
@@ -275,7 +273,14 @@ def get_loop_sleep_time(self):
         """
         return max(0.0, self.sleep_time_s)
 
-    
+    def wait_until_ready_to_run(self,sleep=.005):
+        if self.ts_loop_start is None:
+            time.sleep(.01)
+            return True
+        while time.time()-self.ts_loop_start<(1/self.target_loop_rate):
+            time.sleep(.0005)
```

Comment:
Sleep parameter is not passed.
```suggestion
            time.sleep(sleep)
```